### PR TITLE
[Refactor] Remove constants from store

### PR DESCRIPTION
**Pull request by bjornstar**
These aren't really constants if we are changing them.

We can move `vehicleConfig` to the root of the store.

Also `cameraType` can move out of controls as it's not a key we are using to control the car.

### CODE REVIEW

**Comment by njm222** [comment]
Yes I agree! Thanks :)

File: src/controls/KeyboardControls.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { useEffect } from 'react'
 import { useStore } from '../store'
 
+export const cameraTypes = ['DEFAULT', 'FIRST_PERSON', 'BIRD_EYE']
```
Comment (njm222) [inline src/controls/KeyboardControls.js:4]: I would move this to a types / enums file

**Review by njm222 — CHANGES_REQUESTED** [review]
Looks great, much cleaner and easier to read!

**Comment by njm222** [comment]
There's a few conflicts that still need to resolved

**Comment by njm222** [comment]
Still has issues from the merge in Editor.js

**Comment by bjornstar** [comment]
Would love to get this in before another conflict erupts :)

File: .eslintrc.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
         "react"
     ],
     "rules": {
-        "no-unused-vars": "warn",
+        "no-unused-vars": ["warn", { "ignoreRestSiblings": true }],
```
Comment (njm222) [inline .eslintrc.json:22]: Everything looks good! What is this for?

**Review by njm222 — APPROVED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (bjornstar) [reply .eslintrc.json:22]: https://github.com/pmndrs/racing-game/pull/46/files#diff-1843bac69f436bba37a5448987963889810008090bde459ef5dbd50d46016ae7R4 

```js
const { directionLocal, axleLocal, chassisConnectionPointLocal, rollInfluence, ...filteredWheelInfo } = wheelInfo
```

We use the rest operator to exclude the properties we don't modify. Technically the properties we filter out are unused, but it's a functional way of not using them.

**Pull request merged**